cmd/infra/powervs/create.go

```diff
@@ -59,6 +59,7 @@ const (
 	dhcpServiceErrorState = "ERROR"
 
 	// Time duration for monitoring the resource readiness
+	dhcpPollingInterval              = time.Second * 30
```

Comment:
if this service is taking around ~10mins then I feel 1min can be a suitable polling interval.